web_view_google_map/README.rst

```diff
@@ -0,0 +1,358 @@
+===============
```

Comment:
To avoid the RST lint errors, copy the template at https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst